core/src/banking_stage/forward_packet_batches_by_accounts.rs

```diff
@@ -130,6 +134,10 @@ impl ForwardPacketBatchesByAccounts {
     pub fn iter_batches(&self) -> impl Iterator<Item = &ForwardBatch> {
         self.forward_batches.iter()
     }
+
+    pub fn take_iter_batches(self) -> impl Iterator<Item = ForwardBatch> {
```

Comment:
I send the packets from these ForwardBatch, created by scheduler, to workers. Instead of cloning the packets, just take ownership.